plugin/storage/integration/fixtures/traces/tags_opname_trace.json

```diff
@@ -7,13 +7,13 @@
       "references": [
         {
           "refType": "CHILD_OF",
-          "traceId": "AAAAAAAAAAAAAAAAAAAA/w==",
+          "traceId": "AAAAAAAAAAAAAAAAAAAAEg==",
           "spanId": "AAAAAAAAAP8="
         },
         {
-          "refType": "CHILD_OF",
-          "traceId": "AAAAAAAAAAAAAAAAAAAAAQ==",
-          "spanId": "AAAAAAAAAAI="
+          "refType": "FOLLOWS_FROM",
```

Comment:
Wouldn't this reduce the effectiveness of the test? Jaeger allows multiple child-of refs so that users can model fork-join relationships

Reply:
Your reference to OpenMetrics threw me off a bit, until I clicked on the link that referred to OpenTracing.

Yes, `opentracing.ref_type` is what should be used, it's why it was added to the transformation rules.

Reply:
OMG, my heads a mess right now with the names of things.

Thanks :)